### DISCUSSION TITLE
[rocprim] Update to 6.1?

### DISCUSSION THREAD

**Discussion by Gardene-el** [rocm-arch/rocm-arch#986] [Support] [answered]
The building instruction for [rocm-examples](https://github.com/ROCm/rocm-examples) will failed due to [`error: Illegal instruction detected: Invalid dpp_ctrl value: broadcasts are not supported on GFX10+` ](https://github.com/ROCm/rocm-examples/issues/110), which fixed in [ROCm/rocPRIM#452](https://github.com/ROCm/rocPRIM/issues/452) , and it is only in 6.1.
In conclusion, for some basic feature support, update to 6.1?

**Comment by salman-farooq-sh** [comment]
Eagerly waiting for this as my 680m and 6500m worked perfectly with 6.1 on ubuntu 22.04 but 6.0 on Arch linux doesn't. Also the version right before 6.0 also worked well on Arch linux.

**Comment by acxz** [comment, marked as answer]
rocprim is now part of [extra] here: https://archlinux.org/packages/extra/any/rocprim/

Please report out of date packages and other issues there.
